packages/core/http/core-http-router-server-internal/src/versioned_router/core_versioned_route.ts

```diff
@@ -137,13 +146,13 @@ export class CoreVersionedRoute implements VersionedRoute {
       mutableCoreKibanaRequest.query = {};
     }
 
-    const result = await handler.fn(ctx, mutableCoreKibanaRequest, res);
+    const response = await handler.fn(ctx, mutableCoreKibanaRequest, res);
 
-    if (this.validateResponses && validation?.response?.[result.status]) {
-      const responseValidation = validation.response[result.status];
+    if (this.router.validateResponses && validation?.response?.[response.status]) {
+      const responseValidation = validation.response[response.status];
       try {
         validate(
-          { body: result.payload },
+          { body: response.payload },
           { body: responseValidation.body, unsafe: { body: validation.response.unsafe?.body } },
           handler.options.version
```

Comment:
Again for my info: what's the behavior of 'unsafe'?

Reply:
TBH, not entirely sure :smile:. This seems to apply to some kind of built in validation, I am just passing this through to the underlying router API call.